digitalpublicgoods/provenance-services-with-smart-data-agreements-using-data-exchange-protocols.json

```diff
@@ -0,0 +1,92 @@
+{
+  "name": "Provenance Services with Smart Data Agreements using Data Exchange Protocols",
+  "clearOwnership": {
+    "isOwnershipExplicit": "Unknown",
+    "copyrightURL": ""
+  },
+  "platformIndependence": {
+    "mandatoryDepsCreateMoreRestrictions": "No",
+    "isSoftwarePltIndependent": "",
+    "pltIndependenceDesc": ""
+  },
+  "documentation": {
+    "isDocumentationAvailable": "Yes",
+    "documentationURL": [
+      "https://github.com/decentralised-dataexchange/data-exchange-agreements/tree/main"
+    ]
+  },
+  "NonPII": {
+    "collectsNonPII": "No",
+    "checkNonPIIAccessMechanism": "",
+    "nonPIIAccessMechanism": ""
+  },
+  "privacy": {
+    "isPrivacyCompliant": "Yes",
+    "privacyComplianceList": [
+      "GDPR"
+    ],
+    "adherenceSteps": [
+      "Uses Data Protection Impact Assessment as the basis for creation of any Data Agreement"
+    ]
+  },
+  "standards": {
+    "supportStandards": "Yes",
+    "standardsList": [
+      "ISO27560"
+    ],
+    "evidenceStandardSupport": [
+      ""
```

Comment:
Confirm whether there is any eveidence for ISO27560 support. Thanks.

Reply:
ISO27560 is not public as yet. Its work in progress.

Reply:
@lalc thank you for this information. Considering `ISO27560` is still work in progress. Please share any other open standards adhered to during the design and development. Please refer to our resources for [tips](https://github.com/DPGAlliance/publicgoods-candidates/blob/main/help-center/open-standards.md). Thanks.

Reply:
Thank you @nathanbaleeta. Here is the list:

Security: HTTPS, SSL, SSH, RS256, ED2556, AnonCreds
Supported Authentication & Authorization: OAuth 2, OIDC and JWT (JSON Web Tokens)
Messaging: DIDComm standards: https://didcomm.org/book/v2/
Data exchange supported formats: JSON